docs/authentication.md

```diff
@@ -0,0 +1,30 @@
+# Authentication
+
+## Find
```

Comment:
Worth adding a line here to say that Find is publically available, but certain parts of it are behind basic auth?

Reply:
done